mxtools/raster_flyer.py

```diff
@@ -42,6 +42,35 @@ def configure_detector(self, **kwargs):
         self.detector.file.external_name.put(file_prefix)
         self.detector.file.write_path_template = data_directory_name
 
+    def configure_zebra(self, **kwargs):
+        angle_start = kwargs["angle_start"]
+        exposurePeriodPerImage = kwargs["exposure_period_per_image"]
+        detector_dead_time = kwargs["detector_dead_time"]
+        scanWidth = kwargs["scan_width"]
+        imgWidth = kwargs["img_width"]
+        numImages = kwargs["num_images"]
+        self.zebra_daq_prep()
+        self.zebra.pc.encoder.put(3)  # encoder 0=x, 1=y,2=z,3=omega
+        ttime.sleep(1.0)
```

Comment:
Reason for sleep, if helpful

Reply:
it's historical - a copy of what was done in LSDC 1, and we have not tested removing this. will add a commit that explains this.

Reply:
see https://github.com/NSLS-II-AMX/mxtools/pull/25/commits/4328f65fbeaadaf09a478ff33b215acbbe8549a3